packages/node_modules/@webex/react-container-message-composer/src/container.js

```diff
@@ -177,8 +180,21 @@ export class MessageComposer extends Component {
       } = props;
 
       const files = constructFiles(e.target.files);
-
-      props.addFiles(conversation, activity, files, sparkInstance);
+      const overThreshold = files.some((file) => file.overThreshold);
+
+      if (overThreshold) {
+        props.addError({
+          id: 'file-attachment-threshold',
+          actionTitle: 'Dismiss',
+          onAction: () => props.removeError('file-attachment-threshold'),
+          displayTitle: 'What if I told you',
```

Comment:
Can you add a screenshot on the PR description as to how this alert looks like?

Reply:
<img width="501" alt="Screenshot 2020-08-13 at 3 33 49 PM" src="https://user-images.githubusercontent.com/1916496/90184640-06061800-dd7b-11ea-9614-57abf726c308.png">
<img width="500" alt="Screenshot 2020-08-13 at 3 34 02 PM" src="https://user-images.githubusercontent.com/1916496/90184646-07cfdb80-dd7b-11ea-9d87-c8d3924030ea.png">

Reply:
when you click dismiss it goes back to normal state

Reply:
Thanks for adding the screenshot 👍🏼
Here's my wording suggestion:
```suggestion
          displayTitle: 'Too big for the web!',
```